doc/docs/download_cfd.md

```diff
@@ -10,35 +10,35 @@ See the [Launcher overview](overview_cfd.md) and [Installing nRF Connect for Des
 
 ## Additional requirements
 
-Running nRF Connect for Desktop has the following additional requirements.
+Running nRF Connect for Desktop has the following additional requirements:
 
-### SEGGER J-Link driver
+- **SEGGER J-Link** - Required on all platforms.
 
-Required on all platforms.
+    - On Windows, the driver comes bundled with nRF Connect for Desktop.
+    - On macOS and Linux, you must install the driver manually. Download the installer for your platform from [SEGGER J-Link Software](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).
```

Comment:
Was always curious how users will know what version to download. We do not specify anywhere what version is required by the current app version (nrfutil version we use), so user will see missmatch only after installing some version of the JLink and running an app.

Reply:
https://nordicsemi.atlassian.net/browse/NRFU-1160

Reply:
Will raise discussion if we'll hint uers somehow in our apps (pointing to the version to install based on nrfutil used in app)

Reply:
We do hint users about the version if there is a mismatch. It's in the logs of most apps (except quickstart) on startup as a warninge.

Reply:
Discussed with @chunfantasy, agreed to handle it in the apps and suggesting JLink in the docs will be messy and who would check app's doc after installing launcher?
[NCD-10](https://nordicsemi.atlassian.net/browse/NCD-1043) for the details.
Closing this discussion. This update is good as it is.